rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -425,11 +425,15 @@ namespace rsx
 				const u32 load = rsx::method_registers.transform_constant_load();
 
 				u32 rcount = count;
-				if (const u32 max = (load + reg) * 4 + count + subreg; max > 468 * 4)
+				if (const u32 max = (load + reg) * 4 + count + subreg, limit = 468 * 4; max > limit)
 				{
 					// Ignore addresses outside the usable [0, 467] range
 					rsx_log.warning("Invalid transform register index (load=%u, index=%u, count=%u)", load, index, count);
-					rcount -= max - (468 * 4);
+
+					if (max - count < limit)
```

Comment:
```suggestion
					if ((max - count) < limit)
```

Reply:
I dont like it, its common sense +- operators preceeds comparsions and so its used everywhere even in rpcs3 code.

Reply:
It's not that someone will get stumped thinking hard on it. I never write bare condionals on arithmetic like this, pretty sure its not used in RSX code.

Reply:
Same thing with long ternaries like a > b? c > d? e > f? ....

Reply:
If its long and not written by you it becomes easy to not spot mistakes.

Reply:
alright